src/libsyntax/feature_gate.rs

```diff
@@ -443,6 +446,19 @@ pub fn deprecated_attributes() -> Vec<&'static (&'static str, AttributeType, Att
     BUILTIN_ATTRIBUTES.iter().filter(|a| a.2.is_deprecated()).collect()
 }
 
+pub fn is_builtin_attr<A>(attr: &A) -> bool where A: PartialEq<&'static str> {
```

Comment:
I would think they might eventually want to use `Attribute` or `Ident`, neither of which are `PartialEq<&str>` (afaik).